tools/extra/fpgabist/bist/bist_dc.c

```diff
@@ -265,7 +265,7 @@ int main(int argc, char *argv[])
 	fpga_token	   accelerator_token;
 	fpga_handle	   accelerator_handle;
 	int open_flags = 0;
-
+	uint32_t i = 0;
```

Comment:
This file should go into a separate pull request.

Reply:
removed from merge request